intera_examples/scripts/send_random_trajectory.py

```diff
@@ -0,0 +1,176 @@
+        KinematicTrajectory object in the motion controller. This feature
+        is included for testing new trajectory generation algorithms. In this
+        case, we override the default (set in motion_controller.cpp) and used
+        the CONSTRAINED_CLAMPED_CUBIC architecture instead.
+
+    -o ~/Desktop/fileName.bag
+    --> Save the trajectory message to a bag file
+
+    -p
+    --> Prints the trajectory to terminal before sending
+
+    """
+    arg_fmt = argparse.RawDescriptionHelpFormatter
+    parser = argparse.ArgumentParser(formatter_class=arg_fmt,
+                                     description=main.__doc__)
+    parser.add_argument(
+        "-n", "--waypoint_count", type=int, default=5,
+        help="number of waypoints to include in the trajectory")
+    parser.add_argument(
+        "-d", "--stepDistance", type=float, default=0.05,
+        help="normalized random walk step distance")
+    parser.add_argument(
+        "-b", "--boundaryPadding", type=float, default=0.1,
+        help="normalized padding to apply to joint limits")
+    parser.add_argument(
+        "-t", "--trajType", type=str, default='JOINT',
+        choices=['JOINT', 'CARTESIAN'],
+        help="trajectory interpolation type")
+    parser.add_argument(
+        "--trajArch", type=str, default='',
+        choices=['', 'DEFAULT', 'CONSTRAINED_COMPOSITE_CUBIC',
+                 'CONSTRAINED_BOUNDARY_CUBIC', 'CONSTRAINED_CLAMPED_CUBIC',
+                 'CLAMPED_CUBIC', 'CLAMPED_BOUNDARY_CUBIC'],
+        help="trajectory architecture")
+    parser.add_argument(
+        "-s",  "--speed_ratio", type=float, default=0.5,
+        help="A value between 0.0 (slow) and 1.0 (fast)")
+    parser.add_argument(
+        "-a",  "--accel_ratio", type=float, default=0.5,
+        help="A value between 0.0 (slow) and 1.0 (fast)")
+    parser.add_argument(
+        "-o", "--output_file",
+        help="Save the trajectory task to a bag file")
+    parser.add_argument(
+        "--output_file_type", default="yaml",
+        choices=["csv", "yaml"], help="Select the save file type")
+    parser.add_argument(
+        "-p", "--print_trajectory", action='store_true', default=False,
+        help="print the trajectory after loading")
+    parser.add_argument(
+        "--do_not_send", action='store_true', default=False,
+        help="generate the trajectory, but do not send to motion controller.")
+    parser.add_argument(
+        "--log_file_output",
+        help="Save motion controller log messages to this file name")
+    args = parser.parse_args()
+
+    if args.waypoint_count < 1:
+        args.waypoint_count = 1
+        rospy.logwarn('Input out of bounds! Setting waypoint_count = 1')
+
+    try:
+        rospy.init_node('send_random_joint_trajectory_py')
+
+        # Set the trajectory options
+        traj_opts = TrajectoryOptions()
+        traj_opts.trajectory_architecture_type = args.trajArch
```

Comment:
We may want to remove this feature for right now. I think it will likely confuse users and there is a good chance that it will change in the near future. Maybe leave it in the class, but remove it from the example?